core/rpc/src/lib.rs

```diff
@@ -45,6 +46,9 @@ pub trait MercuryRpc {
 
     #[rpc(name = "get_transaction_history")]
     fn get_transaction_history(&self, ident: String) -> RpcResult<Vec<TransactionWithStatus>>;
+
+    #[rpc(name = "query_charge")]
```

Comment:
I think charge is not an accurate name.  If we want to make this interface common, I suggest query_income, because this name can not only for exchange querying recharge but also for wallet querying income.